### ISSUE TITLE
Errors after update

### ISSUE THREAD

**Issue by Vitalickus** [Rem0o/FanControl.Releases#1656]
Hi, im getting these errors after the update all the time, any fix ? 
![image](https://user-images.githubusercontent.com/128933719/227739608-0db2aae6-74f7-4eb3-976d-98cba048c8d8.png)

**Comment by Rem0o** [comment]
any log.txt?

**Comment by a-shkalyabin** [comment]
The same issue after update/fresh install v152. Also there is no any logged errors at all. Plz let me know if I can assist anyhow.
![Screenshot 2023-03-31 033400](https://user-images.githubusercontent.com/20355937/228994058-597736b4-8fc9-423a-92d3-22eb0d99c6ca.png)
![Screenshot 2023-03-31 033513](https://user-images.githubusercontent.com/20355937/228994066-99012f17-d408-4bac-b91a-a190d9edc2b7.png)

**Comment by Rem0o** [comment]
@a-shkalyabin 
In your case it seems like the driver renamed the fans at index 0,1 and 2, (So fan 1, fan 2 and fan 3). We can see "3 new"  disabled control cards on the right. Those are the equivalent. Run an assisted setup to reset those up, assign the fan curve and you should be good to go.

**Comment by a-shkalyabin** [comment]
I tried to run setup from scratch of course, but result was the same. That's how it looks in v147, to which I've rolled back to.
![image](https://user-images.githubusercontent.com/20355937/229100096-6db40cf9-72b5-431c-b661-e214ba5bab17.png)
BTW, bottom#1 and bottom#2 were never managed to be be controlled by FC, SmartFan always overrides it somehow. Tried many presets and manual mode in bios as well.
